Frontend/src/components/Database/Database.js

```diff
@@ -202,10 +202,26 @@ class Database {
       limit: 10,
       fields: fields,
       selector: selector,
-    });
+    }); 
     return result.docs;
   }
 
+  async fetchUniqueDocsBySelector(selector, fields) {
+    const result = await this.database.find({
+      limit: 9999,
```

Comment:
It also works quite well with 100, this will improve performance

Reply:
true, if one letter has already been supplied chances are big that enough is there to present :)